src/mlpack/methods/ann/layer/atrous_convolution.hpp

```diff
@@ -85,7 +87,24 @@ class AtrousConvolution
                     const size_t inputHeight = 0,
                     const size_t dilationWidth = 1,
                     const size_t dilationHeight = 1,
-                    const std::string& paddingType = "None");
+                    const std::string& paddingType = "none");
+  /**
+   * Create the AtrousConvolution object using the specified number of output
+   * maps, filter size, stride and padding parameter. The number of
+   * input maps will be inferred.
+   *
+   * @param outSize The number of output filter.
+   * @param kernelSize The height and width of the convolution window;
+   * @param strides The height and width of the stride.
+   * @param dilation The space between the cells of filters.
+   * @param padding The type of padding ('valid' or 'same', 'none').
+   *     Defaults to 'none'.
+   */
+  AtrousConvolution(const size_t filter,
```

Comment:
```suggestion
  AtrousConvolution(const size_t outSize,
```
(The parameter name doesn't seem to match the documentation; maybe `filter` or `filters` is a better name, I'm not sure.)